roles/redhat_satellite6_lb/README.md

```diff
@@ -0,0 +1,64 @@
+redhat_satellite6_lb
```

Comment:
since this is specifically capsule loadbalancers can you make the naming be `redhat_satellite6_capsule_lb`